ops/main.py

```diff
@@ -299,7 +298,7 @@ def main(charm_class):
     charm_state_path = charm_dir / CHARM_STATE_FILE
     framework = ops.framework.Framework(charm_state_path, charm_dir, meta, model)
     try:
-        charm = charm_class(framework, None)
```

Comment:
Actually a bigger problem is probably this one, since charms that are using the (framework, key) syntax break on this, as it won't be passing a mandatory argument.

I suppose we could use introspection:
https://docs.python.org/3.8/library/inspect.html#introspecting-callables-with-the-signature-object

To figure out how to give a deprecation warning.

Reply:
It's simpler, just passing not None as I specified above.